docs/topics/logging.txt

```diff
@@ -261,7 +267,29 @@ First, here's a simple configuration which writes all request logging from the
 If you use this example, be sure to change the ``'filename'`` path to a
 location that's writable by the user that's running the Django application.
 
-Second, here's an example of a fairly complex logging setup, configured using
+Second, here's an example of how to make the logging system print everything to
+the console based on the value of an environment variable . It may be useful
+during local development. Set the environment variable
+``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging calls in the console::
+
+    # Not working yet. Entries from django.request and django.security aren't
```

Comment:
Yeah, you're right, I was wrong that this works fully with the default config in place. Sigh. I don't like having all our logging examples bypass Django's logging config system entirely, but perhaps that's really our only accurate option until a proper fix. It seems like both the "console log in dev" example and the "complex" example really should bypass the default config.

It may also be that a "proper fix"  could be as simple as a new `LOGGING_SKIP_DEFAULTS` setting, which could be set to `True` to skip ever installing the default logging config? Then that setting would perform the function that these docs previously assumed `disable_existing_loggers` could perform, and it would simplify the boilerplate needed to do a full override with your own logging config. Instead of setting `LOGGING_CONFIG = None` and then having to import `logging` and call `logging.dictConfig` yourself, all you'd have to do is set `LOGGING_SKIP_DEFAULTS = True` and `LOGGING`.

(Or instead of defining a new setting, we could add our own custom key into `LOGGING`; but I don't like that as well, because it breaks the full equivalence between `LOGGING` and what `dictConfig` is documented to receive.)

I'll make this proposal on the mailing list.